Blockzilla/AppDelegate.swift

```diff
@@ -18,6 +19,32 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             BuddyBuildSDK.setup()
         #endif
 
+        // Set up Telemetry
+        let telemetryConfig = Telemetry.default.configuration
+        telemetryConfig.appName = "Focus"
+        telemetryConfig.dataDirectory = .documentDirectory
+        telemetryConfig.userDefaultsSuiteName = AppInfo.sharedContainerIdentifier
+        
+        telemetryConfig.measureUserDefaultsSetting(forKey: "prefKeyEngine", withDefaultValue: SearchEngineManager(prefs: UserDefaults.standard).engines.first?.name)
+        telemetryConfig.measureUserDefaultsSetting(forKey: SettingsToggle.blockAds.rawValue, withDefaultValue: Settings.getToggle(.blockAds))
```

Comment:
Is there a Swifty way to get rid of the `.rawValue` here? Can we use an `enum`, maybe with a protocol, that these keys need to follow?